src/components/dialog/dialog.css

```diff
@@ -1,4 +1,21 @@
+@import '../app/variablesV2.css';
+@import '../app/mixins.css';
+
 .dialog {
+  border-radius: 6px;
```

Comment:
Also check with Domenico, if we could use the same border as in the other parts of the application.